ge11_team01/models/repair.py

```diff
@@ -0,0 +1,21 @@
+from odoo import api, fields, models
+
+
+class RepairOrder(models.Model):
+    _name = "repair.order"
+    _inherit = ["repair.order", "portal.mixin"]
+
+    is_public = fields.Boolean("Public", default=True)
+
+    def _get_order_portal_return_action(self):
+        self.ensure_one()
+        return self.env.ref("repair.stock_production_lot_view_form")
+
+    def _compute_access_url(self):
+        super()._compute_access_url()
```

Comment:
Do you need the super?

Reply:
Sorry, was copy/paste xD We don't need it :D